openquake/server/static/js/engine.js

```diff
@@ -121,22 +143,22 @@
                 e.preventDefault();
                 var calc_id = $(e.target).attr('data-calc-id');
                 var view = this;
-                diaerror.showDiaError("Removing calculation " + calc_id, "...");
+                diaerror.show("Removing calculation " + calc_id, "...");
                 $.post(gem_oq_server_url + "/v1/calc/" + calc_id + "/remove"
                      ).success(
                          function(data, textStatus, jqXHR)
                          {
-                             diaerror.showDiaError("Removing calculation " + calc_id, "Calculation " + calc_id + " removed.");
+                             diaerror.show("Removing calculation " + calc_id, "Calculation " + calc_id + " removed.");
                              view.calculations.remove([view.calculations.get(calc_id)]);
                          }
                      ).error(
                          function(jqXHR, textStatus, errorThrown)
                          {
                              if (jqXHR.status == 404) {
-                                 diaerror.showDiaError("Removing calculation " + calc_id + ".", "Failed: calculation " + calc_id + " not found.");
+                                 diaerror.show("Removing calculation " + calc_id + ".", "Failed: calculation " + calc_id + " not found.");
                              }
                              else {
-                                 diaerror.showDiaError("Removing calculation " + calc_id + ".", "Failed: " + textStatus);
+                                 diaerror.show("Removing calculation " + calc_id + ".", "Failed: " + textStatus);
```

Comment:
In a title `.` should not be present at the end (remove only from "Removing calculation " + calc_id + ".", not the second statement)